axes/tests/test_views.py

```diff
@@ -0,0 +1,47 @@
+from unittest.mock import Mock, patch
+
+from django.contrib.auth.models import User
+from django.test import TestCase, RequestFactory
+from django.utils.six.moves import http_client
+
+from axes.views import AxesLoginView
+
+
+class AxesLoginViewTest(TestCase):
```

Comment:
Use [override_settings()](https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.override_settings) to use the newer login view logic.